modules/runtime/pages/custom-deployment.adoc

```diff
@@ -6,8 +6,8 @@
 
 [WARNING]
 ====
-Remember that the recommended way of installing Bonita is to use the provided xref:tomcat-bundle.adoc[Tomcat] bundle. +
-It saves you from doing all the following configuration at hand, as the xref:runtime:bonita-platform-setup.adoc[setup tool] included handles it automatically.
+Remember that the recommended way of installing Bonita is through a self-contained application xref:application-build.adoc
```

Comment:
Invalid xref. 
Should be xref:runtime:application-build.adoc
See https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/content/CONTRIBUTING.adoc#adding-reference-between-pages-of-two-components-or-between-pages-of-2-versions-of-the-same-component

Reply:
done :heavy_check_mark: :